client/reader/components/header/index.tsx

```diff
@@ -6,6 +6,7 @@ import clsx from 'clsx';
 import ReaderIcon from 'calypso/assets/icons/reader/reader-icon';
 import { useSelector } from 'calypso/state';
 import { getCurrentUser } from 'calypso/state/current-user/selectors';
+import { HelpCenter } from './help-center';
```

Comment:
Is there any particular reason why we are preferring to copy the MSD components instead of using it from `dashboard/app` directly?

I think we should avoid duplication as this will lead to inconsistencies in future.

```suggestion
import { HelpCenter } from 'calypso/dashboard/app/secondary-menu';
import { recordTracksEvent } from '@automattic/calypso-analytics';
```

And then use it below as following

```typescript
{ user && <HelpCenter user={ user } recordTracksEvent={ recordTracksEvent } /> }
```

In `client/dashboard/app/secondary-menu/index.tsx` we can update the `<Help />` component i.e move`useAuth` & `useAnalytics` hooks to SecondaryMenu component and provide the necessary data from there.

```typescript
function Help() {
```

to

```typescript
interface HelpCenterProps {
	user: User;
	recordTracksEvent: ( eventName: string, properties?: Record< string, any > ) => void;
}

export function HelpCenter( { user, recordTracksEvent }: HelpCenterProps ) {
```

Reply:
Yes, as I explained on your call, we can explore ways to make it shareable and more reusable in a way that we can communicate that changes to these components require more attention.

For now, the idea is to focus more on enhancing the user experience, especially since we still have ongoing UI change issues. Once we learn more, we can consider the best strategy, taking into account cost, risks, and the architecture solutions we want to implement.